lib/porky_lib/symmetric.rb

```diff
@@ -30,6 +30,15 @@ def create_key(tags, key_alias = nil, key_rotation_enabled = true)
     key_id
   end
 
+  def cmk_alias_exists?(key_alias)
+    alias_list = client.list_aliases.to_h[:aliases]
+    alias_list.each do |item|
+      return true if item[:alias_name] == key_alias
+    end
```

Comment:
I think you can also do something like: 
return true if alias_list.key(key_alias)